deepchem/dock/docking.py

```diff
@@ -105,11 +111,15 @@ def dock(self,
       complexes, scores = outputs
     else:
       complexes = outputs
+
     # We know use_pose_generator_scores == False in this case
     if self.scoring_model is not None:
       for posed_complex in complexes:
+        # NOTE: this casting is workaround. This line doesn't effect anything to the runtime
+        self.featurizer = cast(ComplexFeaturizer, self.featurizer)
         # TODO: How to handle the failure here?
-        features, _ = self.featurizer.featurize([molecular_complex])
+        features, _ = self.featurizer.featurize(  # type: ignore
```

Comment:
This line may be a bug.`ComplexFeaturizer` should takes two arguments. 
How should I fix...?

```
# original
self.featurizer.featurize([molecular_complex])

# correct
self.featurizer.featurize([molecular_complex], [posed_complex])

```